==== Integer Arithmetic Bugs ====
SWC ID: 101
Severity: High
Contract: WolvesOfETHSt
Function name: setContractURI(string)
PC address: 13934
Estimated Gas Usage: 23152 - 106633
The arithmetic operator can underflow.
It is possible to cause an integer overflow or underflow in the arithmetic operation. 
--------------------
Initial State:

Account: [CREATOR], balance: 0x0, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , decoded_data: , value: 0x0
Caller: [CREATOR], function: setContractURI(string), txdata: 0x938e3d7b00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000021028001808001082000804000800480010100042000048004200804400002020200, value: 0x0

==== Exception State ====
SWC ID: 110
Severity: Medium
Contract: WolvesOfETHSt
Function name: setContractURI(string)
PC address: 20552
Estimated Gas Usage: 571 - 759
An assertion violation was triggered.
It is possible to trigger an assertion violation. Note that Solidity assert() statements should only be used to check invariants. Review the transaction trace generated for this issue and either make sure your program logic is correct, or use require() instead of assert() if your goal is to constrain user inputs or enforce preconditions. Remember to validate inputs from both callers (for instance, via passed arguments) and callees (for instance, via return values).
--------------------
In file: #utility.yul:1025

panic_error()

--------------------
Initial State:

Account: [CREATOR], balance: 0x0, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , decoded_data: , value: 0x0
Caller: [CREATOR], function: setContractURI(string), txdata: 0x938e3d7b0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000ffffffffffffffe8, value: 0x0

==== Exception State ====
SWC ID: 110
Severity: Medium
Contract: WolvesOfETHSt
Function name: safeTransferFrom(address,address,uint256,bytes)
PC address: 20552
Estimated Gas Usage: 1090 - 1185
An assertion violation was triggered.
It is possible to trigger an assertion violation. Note that Solidity assert() statements should only be used to check invariants. Review the transaction trace generated for this issue and either make sure your program logic is correct, or use require() instead of assert() if your goal is to constrain user inputs or enforce preconditions. Remember to validate inputs from both callers (for instance, via passed arguments) and callees (for instance, via return values).
--------------------
In file: #utility.yul:1031

panic_error()

--------------------
Initial State:

Account: [CREATOR], balance: 0x0, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , decoded_data: , value: 0x0
Caller: [ATTACKER], function: safeTransferFrom(address,address,uint256,bytes), txdata: 0xb88d4fde0000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000003, value: 0x0


